### PR TITLE
Merge tag 3.12.2

### PR DESCRIPTION
**Pull request by phiz71**
gravitee-io/issues#6557

### CODE REVIEW

File: gravitee-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/VirtualHost.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 import io.gravitee.common.http.HttpHeaders;
 import io.gravitee.gateway.api.Request;
+import java.util.Objects;
```
Comment (gcusnieux) [inline gravitee-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/VirtualHost.java:20]: Is this really necessary ?

**Review by gcusnieux — APPROVED** [review]

File: gravitee-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/VirtualHost.java (hunk 1 above)
Comment (jhaeyaert) [reply gravitee-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/VirtualHost.java:20]: No it's not. Can be removed.

Comment (phiz71) [reply gravitee-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/VirtualHost.java]: fixed

**Review by gaetanmaisse — APPROVED** [review]
No idea what I'm approving but we need to merge this one

**Pull request merged**